docs/site/pages/architecture/cross-platform-players.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Cross-Platform-Players'
+---
+
+# Using Player across multiple platforms
+
+One of the major benefits of adopting Player for rendering a UI is it's ability to function across multiple platforms using the same content. In order to facilitate this, and get the most out of Player's architecture, here's a few things to keep in mind as you integrate Player into multiple platforms:
+
+## Use core-plugins to share implementations/functionality
+
+One of the easiest, and most beneficial thing to do when integrating Player is to organize feature sets into plugins. Keeping each plugin small and concise allows for easy sharing of features with other applications.
+
+The core player, as well as each of the platform integration, supplies an interface for users to augment/extend the functionality of Player. This can be used to author validations, formatters, expressions, etc once, and be shared across multiple players.
+
+## What this looks like in practice
+
+Plugins such as the common-types plugin is a great example of a small plugin used to add a very specific feature to the core-player. Since the validations, formats, and data-types present in this plugin are authored once, we can leverage Player's architecture to share these implementations across platforms.
```

Comment:
```suggestion
Plugins such as the common-types plugin is a great example of a small plugin used to add a very specific feature to the Player Core. Since the validations, formats, and data-types present in this plugin are authored once, we can leverage Player's architecture to share these implementations across platforms.
```